ang/civicase/case/details/people-tab/directives/case-details-people-tab.directive.html

```diff
@@ -74,14 +74,45 @@ <h3>{{ ts('%1 Contacts', {1: getCountOfRolesWithContacts(roles)}) }}</h3>
       >
       </paging>
       <div>
-        <a ng-repeat="letter in letters" class="civicase__people-tab__filter-alpha-pager" ng-class="{active: letter === rolesAlphaFilter}" ng-click="setLetterFilter(letter, 'roles')">{{ letter }}</a>
+        <a
+          ng-repeat="letter in letters"
+          class="civicase__people-tab__filter-alpha-pager"
+          ng-class="{active: letter === rolesAlphaFilter}"
+          ng-click="setLetterFilter(letter, 'roles')">
+          {{ letter }}
+        </a>
         <a href class="all" ng-click="setLetterFilter('', 'roles')">All</a>
       </div>
+      <div class="civicase__people-tab__inactive-filter">
+        <span
+          class="civicase__checkbox" role="checkbox" tabindex="0"
+          aria-checked="{{}}" aria-labelledby="Deleted Cases"
```

Comment:
The value of aria-checked looks weird. Is this supposed to be `aria-checked="{{}}"`?